src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1513,6 +1600,7 @@ bool OutputStateMachineEngine::s_ParseColorSpec(const std::wstring_view string,
     }
     return success;
 }
+CATCH_LOG_RETURN_FALSE()

```

Comment:
I had to add this to stop auditing from complaining `substr` and `transform`  will throw exceptions

Reply:
yeah, this is good